ledger/catchpointtracker.go

```diff
@@ -1403,7 +1420,7 @@ func accountHashBuilder(addr basics.Address, accountData basics.AccountData, enc
 }
 
 func (ct *catchpointTracker) catchpointEnabled() bool {
-	return ct.catchpointInterval != 0
+	return ct.catchpointInterval != 0 && ct.renableGeneratingCatchpointFiles
```

Comment:
data race: ct.renableGeneratingCatchpointFiles is updated from another thread. We do not want locks in functions called from commitRound, so make it atomic.